client/src/app/home.js

```diff
@@ -230,29 +105,13 @@ const Home = observer((props) => {
                       10 years, no one gets WET anymore. Ben Shapiro was right.
                     </div>
                     <Box>
-                      <Flex className="about-buttons-container">

```

Comment:
killed the buy waifu button so there's just a single centered learn more button
![image](https://user-images.githubusercontent.com/16842072/110196589-c419ed80-7e02-11eb-96a4-b242ac864c8b.png)